### ISSUE TITLE
Milestone 2 Prep

### ISSUE THREAD

**Issue by arek-kwapis** [Abstrat-Technologies/rustedanvil#22]
Items required to begin on Milestone 2

- [ ] Issues closed
  - [ ] #21 
- [ ] workspaces
  - [ ] local
    - [ ] wipe
    - [ ] reinit
    - [ ] prep
  - [ ] remote
    - [ ] wipe
    - [ ] upgrade ver
    - [ ] redo
    - [ ] init
    - [ ] prep
    - [ ] link
    - [ ] control
- [ ] discord prep
- [ ] final migration

**Comment by Unloaded123456** [comment]
Does this milestone include removing XAMPP implementation and replacing with a more easy-to-use one?

**Comment by arek-kwapis** [comment]
> Does this milestone include removing XAMPP implementation and replacing with a more easy-to-use one?

Yes. XAMPP will be removed in favour of our own custom implementation
